styling/src/jsMain/kotlin/dev.fritz2.styling/theme/default.kt

```diff
@@ -2286,6 +2292,174 @@ open class DefaultTheme : Theme {
+                    override val decorationLeft: Style<BasicParams> = {
+                        css("visibility: hidden")
+                    }
+                    override val decorationTop: Style<BasicParams> = {
+                        css("visibility: hidden")
+                    }
+                }
+            }
+            override val solid: (ColorProperty) -> AlertVariantStyles = {
+                object : AlertVariantStyles {
+                    override val background: Style<BasicParams> = {
+                        background { color { it } }
+                    }
+                    override val text: Style<BasicParams> = {
+                        color { textColorLight }
+                    }
+                    override val accent: Style<BasicParams> = {
+                        color { textColorLight }
+                    }
+                    override val decorationLeft: Style<BasicParams> = {
+                        css("visibility: hidden")
+                    }
+                    override val decorationTop: Style<BasicParams> = {
+                        css("visibility: hidden")
+                    }
+                }
+            }
+            override val leftAccent: (ColorProperty) -> AlertVariantStyles = {
+                object : AlertVariantStyles {
+                    override val background: Style<BasicParams> = {
+                        background { color { alterHexColorBrightness(it, 1.5) } }
+                    }
+                    override val text: Style<BasicParams> = {
+                        color { textColorDark }
+                    }
+                    override val accent: Style<BasicParams> = {
+                        color { it }
+                    }
+                    override val decorationLeft: Style<BasicParams> = {
+                        background { color { it } }
+                    }
+                    override val decorationTop: Style<BasicParams> = {
+                        css("visibility: hidden")
+                    }
+                }
+            }
+            override val topAccent: (ColorProperty) -> AlertVariantStyles = {
+                object : AlertVariantStyles {
+                    override val background: Style<BasicParams> = {
+                        background { color { alterHexColorBrightness(it, 1.5) } }
+                    }
+                    override val text: Style<BasicParams> = {
+                        color { textColorDark }
+                    }
+                    override val accent: Style<BasicParams> = {
+                        color { it }
+                    }
+                    override val decorationLeft: Style<BasicParams> = {
+                        css("visibility: hidden")
+                    }
+                    override val decorationTop: Style<BasicParams> = {
+                        background { color { it } }
+                    }
+                }
+            }
+        }
+    }
+
```

Comment:
I would love to have the bottom and rightAccent variants as well (might not be called variant anymore, tho)